src/SharpGLTF.Cesium/Schema2/CesiumExtensions.cs

```diff
@@ -21,6 +21,8 @@ public static void RegisterExtensions()
             _CesiumRegistered = true;
 
             ExtensionsFactory.RegisterExtension<MeshPrimitive, CesiumPrimitiveOutline>("CESIUM_primitive_outline");
+            ExtensionsFactory.RegisterExtension<MeshPrimitive, MeshExtMeshFeatures>("EXT_mesh_features");
```

Comment:
Think this is not correct MeshPrimitive must be Node/ModelRoot?

Reply:
The extensions need to publish somewhere to which objects in the schema can be attached to, otherwise serialization would fail on load if it finds an extension in the wrong object.